compiler+runtime/include/cpp/jank/runtime/obj/uuid.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <boost/uuid/uuid.hpp>
```

Comment:
I tried to forward declare the type I needed here but couldn't figure out how.

Reply:
We need to store a value of this type in our struct, so we need the definition. We can only get away with a forward decl when we're dealing with only refs and pointers, since they're always the same size.

Reply:
Note that boost has been removed as a dependency. For this, we can use stduuid: https://github.com/mariusbancila/stduuid

We'll need to bring it in as a submodule, under `compiler+runtime/third-party`.